index.html

```diff
@@ -328,7 +328,55 @@ <h3>CONOPY <span>&bull; Back End Dev &bull; 2015</span></h3>
               </div>
             </details>
           </p>
-        </artcle></section
-    ></a>
+        </artcle>
+        <section class="form-wrap-level-one">
+          <div class="form-wrap-level-two">
+            <div class="form-container">
+              <h2 class="form-title">Contact me</h2>
+              <p class="form-description">
+                If you have an application you are interested in developing, a
+                feature that you need built or a project that needs coding. I’d
+                love to help with it!
+              </p>
+              <form action="" method="post" class="contact-form">
```

Comment:
Make sure that the `POST` method of your form is working properly, the user should be able to send a message using  the `action` and `method` attribute  integrated with [formspree service](https://formspree.io/html).

* `action` value should be similar to `https://formspree.io/f/{form_id}`
* `method` value equals `POST`Form uses the action and method attribute to integrate with [formspree service](https://formspree.io/html).

Please, follow the link above and make sure the response is `successful`